dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/sms/EventSavingSMSListener.java

```diff
@@ -318,7 +318,7 @@ private void createAndAddAudit(EventDataValue dataValue, DataElement dataElement
             dataValue.getProvidedElsewhere(),
             ChangeLogType.CREATE);
 
-    dataValueAuditService.addTrackedEntityDataValueChangeLog(dataValueAudit);
+    eventChangeLogService.addTrackedEntityDataValueChangeLog(dataValueAudit);
```

Comment:
I assume this can be removed when use the tracker importer, right? As somewhere in the importer we should be auditing changes to TEs, ...

Reply:
Yes, the importer should end up doing this, I'll change that variable and method name, since they are still called audit...